scheduler/package.json

```diff
@@ -6,6 +6,8 @@
   "private": true,
   "scripts": {
     "start": "nest start --watch",
+    "build": "rm -rf ./dist && nest build",
+    "start:prod": "node dist/main",
```

Comment:
Suggest to use pm2 to protect the node process.

Reply:
can't resolve how to get pm2 log by kubectl logs command for now.  leave it to the next step